crates/next-dev/src/lib.rs

```diff
@@ -286,8 +275,10 @@ async fn source(
     let project_path = fs.root().join(project_relative);
 
     let env = load_env(project_path);
+    let config_output_root = output_fs.root().join(".next/config");
+    let next_config = load_next_config(project_path, config_output_root);
```

Comment:
I think the `web_source` should also be using this, so that we can load the appropriate image config for their external references. (I can't comment on the appropriate line below).

Reply:
Passed it to it too. Not sure if someone uses next/image with CSR mode (non-next), but at least it's consistent now.